test/dlc_tests/container_tests/bin/pytorch_tests/testPyTorch

```diff
@@ -5,5 +5,6 @@ BIN_DIR=${HOME_DIR}/bin
 LOG_DIR=${HOME_DIR}/logs
 
 git clone https://github.com/pytorch/examples.git ${HOME_DIR}/artifacts/examples
+timeout 5m ${BIN_DIR}/pytorch_tests/testPyTorchHelper
```

Comment:
Is timeout sufficient? I prefer it be 300s instead of 5m like in smdebug

Reply:
Turned to fix issue from another aspect